app/controllers/categories_controller.rb

```diff
@@ -0,0 +1,18 @@
+class CategoriesController < ApplicationController
+  DEFAULT_LIMIT = 16
```

Comment:
I think we didn't get to use this constant. Do we still need it :question: :grey_question:  :question: